VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.ts

```diff
@@ -1,24 +1,27 @@
 import { Component, OnInit, ViewChild, ElementRef, Input } from '@angular/core';
-import { HearingSummary } from 'src/app/shared/models/hearing-summary';
 import { ClipboardService } from 'ngx-clipboard';
+import { VhoQueryService } from '../services/vho-query-service.service';
 
 @Component({
     selector: 'app-copy-id',
     templateUrl: './copy-id.component.html'
 })
 export class CopyIdComponent implements OnInit {
-    @Input() conference: HearingSummary;
+    @Input() conferenceId: string;
     tooltip: string;
+    hearingId: string;
     @ViewChild('copyID', { static: false }) copyID: ElementRef;
 
-    constructor(private clipboardService: ClipboardService) {}
+    constructor(private clipboardService: ClipboardService, private vhoQueryService: VhoQueryService) {}
 
-    ngOnInit(): void {
+    async ngOnInit() {
         this.resetText();
+        const response = await this.vhoQueryService.getConferenceByIdVHO(this.conferenceId);
+        this.hearingId = response.hearing_id;
     }
 
-    copyToClipboard(conference: HearingSummary) {
-        this.clipboardService.copyFromContent(conference.id);
+    copyToClipboard() {
+        this.clipboardService.copyFromContent(this.hearingId);
```

Comment:
why are we changing this? This is different to the link we use for the quick link?